saludos.json

```diff
@@ -73,5 +73,6 @@
   "bandolero",
   "genio",
   "valle",
-  "parce"
-]
+  "parce",
+  "panas"
+]
```

Comment:
Missing eof space https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=The%20way%20Unix%20regards%20its,an%20%5Cn%20to%20end%20it.

Reply:
I agree with @druizfly , maybe we have to do as he point @andoniaf

Reply:
Done